client/extensions/woocommerce/app/orders/orders-filter-nav.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionNav from 'components/section-nav';
+
+class OrdersFilterNav extends Component {
+	render() {
+		const { translate, site, status } = this.props;
+		return (
+			<SectionNav>
+				<NavTabs label={ translate( 'Status' ) } selectedText={ translate( 'All orders' ) }>
+					<NavItem path={ getLink( '/store/orders/:site', site ) } selected={ 'any' === status }>
+						{ translate( 'All orders' ) }
+					</NavItem>
+					<NavItem path={ getLink( '/store/orders/processing/:site', site ) } selected={ 'processing' === status }>
+						{ translate( 'Processing' ) }
+					</NavItem>
+					<NavItem path={ getLink( '/store/orders/completed/:site', site ) } selected={ 'completed' === status }>
+						{ translate( 'Completed' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( '&laquo; completed' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/orders-status-filtering)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).